web/sites/default/config/default/core.entity_form_display.eventseries.default.default.yml

```diff
@@ -112,6 +113,12 @@ content:
       placeholder_url: ''
       placeholder_title: ''
     third_party_settings: {  }
+  field_skill_level:
+    type: options_select
```

Comment:
We should make this a multiselect field to accommodate trainings that are suitable for multiple skill levels.

Reply:
I made the change locally, and it has a side effect that the event gets listed N times in the events lists,  where N is  the number of skill level items selected in all the events lists.  Do you want me to keep this change and try to figure out why that happens?

Reply:
There is a setting on the view to combine multiple instances. 
I think the multiple setting might make the filter not work anymore though.

Reply:
I changed the settings on the new filter. Seems to work, tested with various combos of filters